src/components/page/PostSection.tsx

```diff
@@ -0,0 +1,158 @@
+  @media (max-width: 576px) {
+    .post-card-container:nth-child(2) {
+      display: none;
+    }
+  }
+  @media (max-width: ${BREAK_POINT}px) {
+    .post-card-container:nth-child(3) {
+      display: none;
+    }
+  }
+  @media (min-width: ${BREAK_POINT}px) {
+    margin-bottom: 0;
+    width: 100%;
+  }
+`
+const StyledPostAbstract = styled.div`
+  @media (min-width: ${BREAK_POINT}px) {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+    overflow: hidden;
+    color: var(--gray-darker);
+    line-height: 1.69;
+    letter-spacing: 0.4px;
+    text-align: justify;
+  }
+`
+const StyledPostMeta = styled.div`
+  color: var(--gray-dark);
+  font-size: 14px;
+  letter-spacing: 0.4px;
+
+  i,
+  span {
+    line-height: 20px;
+  }
+  @media (min-width: ${BREAK_POINT}px) {
+    > div {
+      display: inline;
+    }
+  }
+`
+
+const PostSection: React.FC<{ options: any }> = ({ options }) => {
+  const { loadingPosts, posts, errorPosts } = useLatestPost({ limit: 3 })
+
+  if (loadingPosts || errorPosts)
+    return (
+      <div className="container mb-5">
+        <Skeleton height="20px" my="10px" />
+        <Skeleton height="20px" my="10px" />
+        <Skeleton height="20px" my="10px" />
+      </div>
+    )
+
+  if (posts.length === 0) return null
+
+  return (
+    <StyledSection className="page-section">
+      <SectionTitle>{options?.title || '部落格專欄'}</SectionTitle>
+
+      <div className="container px-0">
+        <StyledPostListContainer className="mb-5">
+          {posts
+            .sort((a, b) => (a.publishedAt && b.publishedAt ? b.publishedAt.getTime() - a.publishedAt.getTime() : 1))
+            .map(post => (
+              <div key={post.id} className="col-12 col-md-6 col-lg-4 mb-4 post-card-container">
+                <StyledPostCard className="mb-4" to={`/posts/${post.codeName || post.id}`}>
+                  <StyledPostCoverContainer className="mb-3">
+                    <StyledCover>
+                      {typeof post.videoUrl === 'string' && (
+                        <StyledVideoIconBlock>
+                          <PlayCircleIcon />
+                        </StyledVideoIconBlock>
+                      )}
+                      <StyledCustomRatioImage width="100%" ratio={2 / 3} src={post.coverUrl || EmptyCover} />
+                    </StyledCover>
+                  </StyledPostCoverContainer>
+                  <StyledPostTitle className="list-item">{post.title}</StyledPostTitle>
+                  <StyledPostMeta className="mb-2">
+                    <CalendarAltOIcon className="mr-1" />
+                    <span>{post.publishedAt ? moment(post.publishedAt).format('YYYY-MM-DD') : ''}</span>
+                  </StyledPostMeta>
+                  <StyledPostAbstract>
+                    {BraftEditor.createEditorState(post.description)
+                      .toHTML()
+                      .replace(/(<([^>]+)>)/gi, '')
+                      .substr(0, 50)}
```

Comment:
NITS: `.substr()` is an old standard function, use `.substring()` or `.slice()` instead

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr